components/collective-page/BudgetStats.js

```diff
@@ -82,23 +82,28 @@ const BudgetStats = ({ collective, stats }) => {
       <StatContainer>
         <StatTitle>
           <ShowChart size="12px" />
-
-          <DefinedTerm
-            term={Terms.TOTAL_RAISED}
-            textTransform="uppercase"
-            color="black.700"
-            extraTooltipContent={
-              <Box mt={2}>
-                <FormattedMessage
-                  id="budgetSection-raised-total"
-                  defaultMessage="Total contributed before fees: {amount}"
-                  values={{
-                    amount: formatCurrency(stats?.totalAmountRaised.valueInCents || 0, collective.currency, { locale }),
-                  }}
-                />
-              </Box>
-            }
-          />
+          {collective.id === collective.host.id ? (
```

Comment:
Can we use `collective.type` here?

Reply:
I've altered the condition to use `collective.type === CollectiveType.ORGANIZATION`.

Reply:
I think there's a boolean flag `collective.isHost` there.

Reply:
Yes, seems I've missed that. That makes it even simpler. Done. 👍🏽